src/is_wire/rpc/context.py

```diff
@@ -0,0 +1,12 @@
+class Context(object):
```

Comment:
Add `span_name` property in order to be able to chose the span name on service provider handlers.

```python
from six import string_types
from ..core.utils import assert_type

class Context(object):
    def __init__(self, request, reply):
        self._request = request
        self._reply = reply
        self._span_name = None

    @property
    def request(self):
        return self._request

    @property
    def reply(self):
        return self._reply
    
    @property
    def span_name(self):
        return self._span_name
    
    @span_name.setter
    def span_name(self, span_name):
        assert_type(span_name, string_types, "span_name")
        self._span_name = span_name

    def has_span_name(self):
        return bool(self._span_name)
```